src/prefs.ts

```diff
@@ -125,6 +132,11 @@ function settings_dialog_view(): [AppWidgets, Gtk.Container] {
         hexpand: true
     })
 
+    const stack_tab_buttons_label = new Gtk.Label({
+        label: "Show Tab Buttons for Stacked Windows",
```

Comment:
Please rename the label here to `Show tabs for stacked windows` to match the gschema description.

Reply:
@jacobgkau done